src/main/java/envoy/server/data/package-info.java

```diff
@@ -0,0 +1,9 @@
+/**
+ * This package contains all (wrapper) data objects needed for Envoy Server to
+ * store objects into the database.
```

Comment:
Missing double `<br>` tags.

Also: 
```suggestion
 * store objects inside the database.
```